### DISCUSSION TITLE
Inject a dynamic value into InputType

### DISCUSSION THREAD

**Discussion by abdelrazzaq-dev** [MichalLytek/type-graphql#929] [Q&A]
Hi, I need to track who create what across many entities, is there any way to inject for example a createdBy property into the input type dynamically so I don't have to add it, manually when creating new data, especially when the object is a sub-document array which needs to iterate over all entries.

**Comment by MichalLytek** [comment]
There's no injecting magic. All you can do is to create a custom parameter decorator that will extract current user data from the context, so you can merge it with your args and pass to the inner layers (like service).
